### PR TITLE
Translate the Event Subscription Page into portuguese

### PR DESCRIPTION
**Pull request by Francisco-dos-Santos**
This PR is a translation work

## Changes

 - Translate of the Event Subscription Page into portuguese issue [https://github.com/chaoss/chaoss-slack-bot/issues/89](https://github.com/chaoss/chaoss-slack-bot/issues/89)

## Checklist
schoost

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

Fixes #

### CODE REVIEW

File: Translations/Portuguese/Event Subscriptions.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
 This is the link  to the Event Subscriptions to be translated.
```
Comment (aosccode) [inline Translations/Portuguese/Event Subscriptions.md:1]: @Francisco-dos-Santos por favor, remova essa linha :)

**Review by aosccode — COMMENTED** [review]
@Francisco-dos-Santos Thanks for your contribuition. We are reviewing this PR and we will get back to you soon :) Meanwhile you can explore more issues to fix under this repository or others participating in Hacktoberfest.

File: Translations/Portuguese/Event Subscriptions.md (hunk 1 above)
Comment (Francisco-dos-Santos) [reply Translations/Portuguese/Event Subscriptions.md:1]: Obrigado  farei   isso ainda hoje.@aosccode

**Review by Francisco-dos-Santos — COMMENTED** [review]
update and edit the page event subscription

File: Translations/Portuguese/Event Subscriptions.md (hunk 1 above)
Comment (Francisco-dos-Santos) [reply Translations/Portuguese/Event Subscriptions.md]: update ready.@aosccode

**Comment by Francisco-dos-Santos** [comment]
Já  remove ***@***.***

A sexta, 20/10/2023, 18:48, Angola Open-source Community <
***@***.***> escreveu:

File: Translations/Portuguese/Event Subscriptions.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
-This is the link  to the Event Subscriptions to be translated.
-[Event Subscriptions](https://github.com/chaoss/chaoss-slack-bot/wiki/Bot-Token-Scopes-&-Event-Subscriptions)
\ No newline at end of file
+
```
Comment (antonio-pedro99) [inline Translations/Portuguese/Event Subscriptions.md:1]: @Francisco-dos-Santos por favor remove essa linha

File: Translations/Portuguese/Event Subscriptions.md [hunk 3]
```diff
@@ -1,2 +1,39 @@
...
+
```
Comment (antonio-pedro99) [inline Translations/Portuguese/Event Subscriptions.md:2]: @Francisco-dos-Santos por favor, remova essa linha

File: Translations/Portuguese/Event Subscriptions.md [hunk 4]
```diff
@@ -1,2 +1,39 @@
...
+># [Escopos de token do bot](https://github.com/chaoss/chaoss-slack-bot/wiki/Bot-Token-Scopes-&-Event-Subscriptions#bot-token-scopes)
+
+>Escopos que controlam o que seu aplicativo pode acessar. Esta é uma lista de escopos habilitados 
+para bot CHAOSS Slack. Uma lista abrangente de escopos de permissão está disponível em [api.slack.com/scopes.](https://api.slack.com/scopes)
+
+> |OAuth de Escopo | Descrição|
+> |------------|-----------
+> [app_mentions:read](https://api.slack.com/scopes/app_mentions:read)|Visualize mensagens que mencionam diretamente @welcometochaoss nas conversas em que o aplicativo está
```
Comment (antonio-pedro99) [inline Translations/Portuguese/Event Subscriptions.md:10]:
```suggestion
>[app_mentions:read](https://api.slack.com/scopes/app_mentions:read)|Visualize mensagens que mencionam diretamente @welcometochaoss nas conversas em que o aplicativo está
```

**Review by antonio-pedro99 — APPROVED** [review]
@Francisco-dos-Santos, thanks for your hard work on this.
I left some comments.

File: Translations/Portuguese/Event Subscriptions.md (hunk 3 above)
Comment (Francisco-dos-Santos) [reply Translations/Portuguese/Event Subscriptions.md:2]: Farei  Obrigado.@ aosccode

File: Translations/Portuguese/Event Subscriptions.md (hunk 4 above)
Comment (aosccode) [reply Translations/Portuguese/Event Subscriptions.md:10]: @Francisco-dos-Santos alguma novidade?

Comment (Busayo-ojo) [reply Translations/Portuguese/Event Subscriptions.md:10]: Hi @aosccode, has this been completed?

**Review by antonio-pedro99 — APPROVED** [review]
LGTM

**Comment by Busayo-ojo** [comment]
Hi @antonio-pedro99,

Could you kindly confirm whether this has been completed? If so, I would appreciate your assistance in guiding the contributor through the DCO sign-off process, so I can close this task.

**Comment by antonio-pedro99** [comment]
> Hi @antonio-pedro99,
> 
> Could you kindly confirm whether this has been completed? If so, I would appreciate your assistance in guiding the contributor through the DCO sign-off process, so I can close this task.

Yes, completed. 
I will get in touch with him

**Comment by misspee007** [comment]
Hi! In case you missed it, this has been resolved [here](https://github.com/chaoss/chaoss-slack-bot/pull/111). Thank you for your hard work.

**Pull request closed without merging**